spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -270,7 +270,7 @@
     end
 
     context 'with nil in_works_ids' do
-      let(:parent) { FactoryGirl.create(:generic_work) }
+      let(:parent) { FactoryGirl.create_for_repository(:work) }
```

Comment:
For consistency in this spec file, should the `FactoryGirl.` be removed or is that a different concern/PR?

Reply:
I can change it if you want.

Reply:
I think if I care enough I'll make a PR later to make this consistent across the test suite.